HackIllinois/FlowControllers/HITabBarController.swift

```diff
@@ -41,10 +41,17 @@ class HITabBarController: UITabBarController {
     }
 
     @objc private func codePopupButtonPressed(_ sender: UIButton) {
-        let codePopup = HICodePopupViewController()
-        codePopup.modalPresentationStyle = .overCurrentContext
-        codePopup.modalTransitionStyle = .crossDissolve
-        self.present(codePopup, animated: true, completion: nil)
+        if HIApplicationStateController.shared.isGuest {
+            let notEnabledAlert = UIAlertController(title: "This feature hasn't been enabled yet!", message: "", preferredStyle: .alert)
```

Comment:
Maybe change the text to "You need to log out of your current account and log in as an attendee to check in to events."